lib/view/pages/game_library/game_library_page.dart

```diff
@@ -210,7 +206,7 @@ class _GameLibraryState extends State<GameLibrary> {
                         builder: (BuildContext context) => StatefulBuilder(
                           builder: (context, setState) => AlertDialog(
                             title: const Text('Sort by'),
-                            content: SingleChildScrollView(child: Column(
```

Comment:
And perhaps leave this in as well? Because there was an overflow issue or something and I asked @pieryvdl to fix it I think

Reply:
I tried to remove this, but it broke the whole page...

Reply:
what exactly broke?

Reply:
The page is empty, which means something has than infinite height